.github/workflows/ant.yml

```diff
@@ -0,0 +1,22 @@
+name: Java CI
+
+on: 
+  pull_request:
+    branches:
+    - anshum/test-github-precommit-action
```

Comment:
I don't know if this is strictly necessary and hopefully checkouts happen on the branch if there is a workflow there. Otherwise would be a pain to keep updating this.

Reply:
this was an attempt to not run in master until we knew it is working as expected